autogpt_platform/backend/backend/copilot/dream/batch_callbacks.py

```
@@ -343,17 +387,23 @@ async def handle_dream_batch_result(
+                )
             except Exception:
                 logger.exception(
-                    "Dream batch lock release failed for user=%s", user_id[:12]
+                    "Dream batch lock release failed for user=%s",
+                    authoritative_user_id[:12],
                 )
 
 
 async def _handle_phase_result(
     *,
     rows: list["BatchResultRow"],
```

Ruff (0.16.1)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)